.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 3
```

Comment:
does it update for all branches ? if so, should we avoid for released branches ? [just asking ]

Reply:
Good question. Currently it checks only the default branch, but [it can be configurable](https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#target-branch) if we start using a development branch process in the future.